app/server.py

```diff
@@ -18,6 +18,46 @@ def index():
 ######################################################################
 @app.route('/shopcarts/<int:id>', methods=['GET'])
 def get_shopcarts(id):
+    """
+    Get a Shopcart
+    This endpoint will get a shopcart.
+    ---
+    tags:
+      - Shopcarts
+    produces:
+      - application/json
+    parameters:
+      - name: user_id
+        in: path
+        description: User id shopcart to retrieve
+        type: integer
+        required: true
+    responses:
+      200:
+        description: Shopcart found
+        schema:
+          id: Shopcart
+          properties:
+            user_id:
+              type: integer
+              description: user id
+              required: true
+              default: 1
+            products:
+              type: dict
+              description: products in the shopcart
+              default: {
+                "2": {
```

Comment:
doesn't this return a list now. Your and @radhikamattoo docs look very different. I'm not sure which is the right way though.

Reply:
This shows an example of a possible response, wasn't used in the prof's code so I didn't use it. I'm not sure about that returning a list tho

 I do think that the `products` property should have `required: true` as well, because we return an empty list of products if the shopcart has none.

Reply:
Yeah I followed the code from swagger hub it has more things. The code in the repo is very basic.